mmcv/utils/registry.py

```diff
@@ -45,7 +179,42 @@ def get(self, key):
         Returns:
             class: The corresponding class.
         """
-        return self._module_dict.get(key, None)
+        scope, real_key = self.split_scope_key(key)
+        if scope is not None:
+            if scope in self._children:
```

Comment:
What if users define two registries in one codebases and make one registry be the parent of the other?

```python
A = Registry('models')
B = Registry('sub_models', parent=A)
```

Reply:
The scope of B could be inferred automatically.